### ISSUE TITLE
PLP Drawer item missing `aria-disabled` attribute when disabled

### ISSUE THREAD

**Issue by guywillis** [adaptlearning/adapt-contrib-pageLevelProgress#257]
### Subject of the issue
The PLP Drawer item is missing the `aria-disabled` attribute when disabled. Whilst the `aria-label` attribute is prepended with 'locked' text to signify the button is unavailable, the button itself does not programmatically tell the user it is disabled. 

### Your environment
* Master AT & FW

### Expected behaviour
A `aria-disabled` attribute is added to the PLP Drawer items when they are disabled.
